cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3189,6 +3189,39 @@ action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` s
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Sophos
 
+[Sophos - Potentialy Unwated Application PUA]
```

Comment:
Title should be **[Sophos - Failed to CleanUp Potentialy Unwated Application by Sophos]**

Reply:
Potentially
Unwanted